src/window.js

```diff
@@ -389,25 +415,27 @@ export default function Window({ application }) {
     application,
   });
 
-  async function confirmDiscard() {
-    if (!settings.get_boolean("has-edits")) return true;
-    const agreed = await confirm({
-      transient_for: application.get_active_window(),
-      text: _("Are you sure you want to discard your changes?"),
-    });
-    if (agreed) {
-      settings.set_boolean("has-edits", false);
-    }
-    return agreed;
-  }
-
   const text_decoder = new TextDecoder();
   async function openFile(file) {
     const language = getLanguageForFile(file);
-    if (!language) return;
-
-    const agreed = await confirmDiscard();
-    if (!agreed) return;
+    if (!language) {
+      const toast = new Adw.Toast({
+        title: "File is not loadable",
```

Comment:
```suggestion
        title: _("Unknown file type"),
```

Reply:
@bertob wdyt?

Reply:
Sounds a bit odd, maybe "This file can not be loaded"?

Reply:
"This file can not be loaded" sounds good to me